experiments/conductor/cmd/runner/controller_commands.go

```diff
@@ -334,7 +335,7 @@ The file should follow these requirements:
 Use ReadFile to read the CRD file.
 Use CreateFile to write the YAML content to the %s/create.yaml file.
 Respond only with the YAML content, no explanations.`,
-		testDir, branch.Group, branch.Kind, currentYear, crdGroup, crdVersion, branch.Kind, strings.ToLower(branch.Kind), testDir)
+		testDir, branch.Group, branch.Kind, currentYear, crdGroup, crdVersion, branch.Kind, strings.ToLower(branch.Kind), opts.testDirSuffix, testDir)
```

Comment:
This sort of positional field substitution seems very error prone. Having said that your not making it worse but at some point later it would be nice to clean this up.

Reply:
^ Actually, I only cleaned up the newly added prompts but not the existing ones. Will clean it up later.